### ISSUE TITLE
"Could not generate artifact name for document" if schemaPath is inside src directory

### ISSUE THREAD

**Issue by darrylyeo** [HoudiniGraphql/houdini#586]
### Describe the bug

I customized the `schemaPath` to be inside of my project's `src` folder, and houdini tried to generate artifacts for the generated `.graphql` schema file (thinking it was part of my Svelte project). It threw the following error and caused the build process to fail:

```
❌ Encountered error in src/schema.graphql
Could not generate artifact name for document [the entire contents of my schema]
```

Since my project only uses inline fragments in `.svelte` files, I'm able to mitigate this by modifying `include` to not check for `.graphql` files. Maybe Houdini should exclude the `schemaPath` file automatically while generating the artifacts (or there should be a warning about this in the docs).

Side note: I think printing the file name instead of the entire document contents in error messages would make for a better debugging experience. The schema I was working with was pretty long so I had to scroll a lot to see what was happening 😅 

### Severity

serious, but I can work around it

### Steps to Reproduce the Bug

Change the `schemaPath` in `houdini.config.js` to be inside `src`:
```js
const config = {
   // ...
   schemaPath: './src/schema.graphql',
}
```

Build the SvelteKit project - throws error.

**Comment by AlecAivazis** [comment]
Thanks for opening this @darrylyeo! I'll get the fix deployed this weekend

**Comment by AlecAivazis** [comment]
This should now be fixed on `0.16.8` 👍

**Comment by darrylyeo** [comment]
Awesome work, thank you!
